src/wallet/ismine.h

```diff
@@ -17,13 +17,13 @@ class CScript;
 /** IsMine() return codes */
 enum isminetype : unsigned int
 {
-    ISMINE_NO         = 0,
-    ISMINE_WATCH_ONLY = 1 << 0,
-    ISMINE_SPENDABLE  = 1 << 1,
-    ISMINE_USED       = 1 << 2,
-    ISMINE_ALL        = ISMINE_WATCH_ONLY | ISMINE_SPENDABLE,
-    ISMINE_ALL_USED   = ISMINE_ALL | ISMINE_USED,
-    ISMINE_ENUM_ELEMENTS,
+    ISMINE_NO         = 0,                                    //!< the scriptPubKey is not in the wallet
+    ISMINE_WATCH_ONLY = 1 << 0,                               //!< the scriptPubKey has been imported into the wallet
+    ISMINE_SPENDABLE  = 1 << 1,                               //!< the scriptPubKey corresponds to an address owned by the wallet user (can spend with the private key)
```

Comment:
This should note that needing a private key is only for legacy wallets.

It would be good to mention somewhere that `DescriptorScriptPubKeyMan` (and future `ScriptPubKeyMan` implementations) will only use `ISMINE_NO` and `ISMINE_SPENDABLE`, essentially treating this as a boolean (except for ISMINE_USED which is kind of orthogonal to the other isminetypes).